models/courses/most_recent_course_blocks.sql

```diff
@@ -0,0 +1,37 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_EVENT_SINK_DATABASE", "event_sink"),
+        engine=get_engine("ReplacingMergeTree()"),
+        order_by="(location)",
+        post_hook="OPTIMIZE TABLE {{ this }} {{ on_cluster() }} FINAL",
+    )
+}}
+
+select
+    location,
+    display_name as block_name,
```

Comment:
something going on with this, still looking into it 
![image](https://github.com/user-attachments/assets/becbeff9-8e98-4ff9-bbbf-ab28a5594886)

Reply:
fixed in https://github.com/openedx/tutor-contrib-aspects/pull/907